critiquebrainz/frontend/templates/moderators/moderators.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Moderators') }} - CritiqueBrainz{% endblock %}
+
+{% block content %}
+<div class="container">
+  <h1>Moderators</h1>
+  <div id="list">
+    {% for item in db_admins %}
```

Comment:
Better settle for one way of calling users in a template to avoid confusion. I'd suggest `moderator`/`moderators` since that's what they are called publicly.